docs/how-to-add-a-provider.md

```diff
@@ -254,24 +254,10 @@ These are hardware-specific features that vary by instance configuration, expres
 - `Preemptible`: Whether this instance type supports spot/preemptible pricing
 
 ### 3. Instance Capabilities
-These are runtime state-dependent features for individual running instances. For example, a running instance might support certain operations that a stopped instance cannot, or vice versa. These are typically checked dynamically based on the instance's current `LifecycleStatus`.
-
-### Instance Type Mapping Complexity
-
-Mapping provider-specific instance types to the v1.InstanceType format is one of the trickier aspects of provider integration. You'll need to:
-
-- Parse provider API responses (often with inconsistent formats)
-- Extract GPU specifications from description strings or structured data
-- Convert pricing information to standardized currency amounts  
-- Map provider-specific storage and networking details to v1 fields
-- Generate stable IDs using `MakeGenericInstanceTypeID` if the provider doesn't offer them
-- Handle availability and regional variations consistently
-
-See `convertLambdaLabsInstanceTypeToV1InstanceType()` in ../internal/lambdalabs/v1/instancetype.go for a complete example of this complex mapping process.
+These are runtime state-dependent features for individual running instances, similar to Instance Type capabilities but applied to a running instance rather than the type template. For example, a running instance might support certain operations that a stopped instance cannot, or vice versa. These are typically checked dynamically based on the instance's current `LifecycleStatus`.
```

Comment:
technically true that they can be changed based on state (they are note, good thing to note). however what I'm referring to is some booleans like (stoppable) are replicated on the instance even though you can figure it out through the instance type thats correlated with it. Its redundant but important to remember they need to be accurate none the less. Take a look at the instance